integrations/java-iceberg-1.5/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -0,0 +1,579 @@
+package com.linkedin.openhouse.javaclient;
```

Comment:
Is code duplication the only way? Have we considered inheritance to share the code?

Reply:
Removed the duplicated code.